certified-connectors/MongoDB/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1421 @@
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "dataSource": {
+                  "type": "string",
+                  "description": "datasource",
+                  "title": ""
+                },
+                "database": {
+                  "type": "string",
+                  "description": "database"
+                },
+                "collection": {
+                  "type": "string",
+                  "description": "collection"
+                },
+                "filter": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    }
+                  },
+                  "description": "filter"
+                }
+              },
+              "default": {
+                "dataSource": "{{DATA_SOURCE}}",
+                "database": "{{DATABASE}}",
+                "collection": "{{COLLECTION}}",
+                "filter": {
+                  "name": "John Sample"
+                }
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A JSON object containing the document that is fetched",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "document": {
+                  "type": "object",
+                  "properties": {
+                    "_id": {
+                      "type": "string",
+                      "description": "_id"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "age": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "age"
+                    }
+                  },
+                  "description": "document"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {},
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {},
```

Comment:
@DianaMDB empty schemas are [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).